markup/edit-event.html

```diff
@@ -217,7 +217,7 @@ <h4 class="visually-hidden">Offers:</h4>
                         </label>
                         <input class="event__input  event__input--destination" id="event-destination-1" type="text" name="event-destination" value="Saint Petersburg" list="destination-list-1">
                         <datalist id="destination-list-1">
-                          <option value="Amsterdam"></option>
+                          <option value="Amsterdam"></option>form
```

Comment:
```suggestion
                          <option value="Amsterdam"></option>
```

Таких ошибок можно избежать если самой просматривать содержимое пулл-реквеста перед отправкой.